src/components/tooltip/Tooltip.tsx

```diff
@@ -219,6 +219,12 @@ class Tooltip extends React.Component<TooltipProps, State> {
             theme,
         } = this.props;
 
+        const tooltipText =
+            text && typeof text === 'object' ? (text as React.ReactElement).props.defaultMessage : String(text);
```

Comment:
Instead of accessing `props.defaultMessage` (assuming this test is a `react-intl` MessageDescriptor) you can use `intl.formatMessage` to render the message as a string.